stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/ui/utils/GiphySizingMode.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.utils
+
+/**
+ * Sets the way in which Giphy container
+ * sizing is determined.
+ */
+public enum class GiphySizingMode {
```

Comment:
is this pulled from the compose PR? because we don't want duplication, these things should be reused

Reply:
This was the original but it's was copied into Compose and then re named a bit in compose (it will be here too as I like adaptive more). The removal of SDK specific kdoc commentary will be reflected here too so there won't be any duplication in the end, both will use the same one as is.